lib/Cleantalk/Common/Firewall/FirewallUpdater.php

```diff
@@ -165,13 +167,18 @@ public function update()
 
                     // Do writing to the DB
                     reset( $lines );
+                    $subnets_count_from_stats = isset($helper::getFwStats()['firewall_subnets_count'])
+                        ? $helper::getFwStats()['firewall_subnets_count']
+                        : 0;
+                    $subnets_count = $subnets_count_from_stats ?: 0;
```

Comment:
It seems the variable `$subnets_count_from`_stats is redundant

Reply:
Yes, fixed